### DISCUSSION TITLE
Ikea E2001 downgrade to 2.4.5

### DISCUSSION THREAD

**Discussion by HFlor** [Koenkk/zigbee2mqtt#23363] [Devices]
Hello,

Unfortunately I updated the switch to version 2.4.11. Now the switches work for about an hour, then the commands from the brightness control only come in now and then. If I remove the battery and put it back in, it works for another hour...

How do I get the working firmware 2.4.5 back on the devices? I have already downloaded the file with the software (10078031-zingo_kt_styrbar_remote-2.4.5.ota.ota.signed).

Hardy

**Comment by EricV-hue** [comment]
See  here [https://github.com/Koenkk/zigbee2mqtt/discussions/23243](url)

**Reply by HFlor** [reply]
@EricV-hue Thank you very much, I will try it this evening. I also found the link to the IKEA page for 2.4.5:

http://fw.ota.homesmart.ikea.net/global/GW1.0/01.20.065/bin/10078031-zingo_kt_styrbar_remote-2.4.5.ota.ota.signed
